frontend/src/metabase-types/api/logger.ts

```diff
@@ -0,0 +1,36 @@
+export type LogLevel =
+  | "off"
+  | "fatal"
+  | "error"
+  | "warn"
+  | "info"
+  | "debug"
+  | "trace";
+
+export type TimeUnit =
```

Comment:
Let's call it something specific to logging to not be used anywhere else. Like `LogTimeUnit` or whatever

Reply:
ed2669751d6cfada1f50802e0a3724fcf14b5af5